core/src/main/resources/hudson/model/Run/new-build-page.jelly

```diff
@@ -33,15 +33,18 @@ THE SOFTWARE.
       <script src="${resURL}/jsbundles/pages/job.js" type="text/javascript" defer="true" />
 
       <j:set var="controls">
-        <t:editDescriptionButton permission="${it.UPDATE}"/>
         <l:hasPermission permission="${it.UPDATE}">
           <st:include page="logKeep.jelly" />
         </l:hasPermission>
       </j:set>
 
       <t:buildCaption controls="${controls}">${it.displayName}</t:buildCaption>
 
-      <l:details-bar it="${it}" />
+      <l:details-bar it="${it}">
+        <t:editDescriptionButton permission="${it.UPDATE}"
```

Comment:
Would it be better to either:

1. Create a new tag for this?
2. Pass a feature flag through? - explicitly probably as not all pages can handle it
3. Pass a version through?

I think 2 is probably best.

Not a huge fan of having to pass the class name through for a 'standard' control it should be internalised imo

Reply:
> Pass a feature flag through? - explicitly probably as not all pages can handle it

Something like `compact`?

Reply:
Great - done.